app/cdap/components/ParsingDrawer/index.tsx

```diff
@@ -93,18 +95,18 @@ export default function({ setLoading, updateDataTranformation }) {
   const handleChange = (value: string | boolean, property: string) => {
     setProperties((prev) => ({
       ...prev,
-      [property]: value,
+      [property]: property === 'format' ? (value as string).toLowerCase() : value,
     }));
   };
 
   const componentToRender = (
     <DrawerWidget
-      headingText={T.translate('features.WranglerNewParsingDrawer.parsing')}
+      headingText={T.translate('features.WranglerNewUI.WranglerNewParsingDrawer.parsing')}
       openDrawer={setDrawerStatus}
       showDivider={true}
       headerActionTemplate={
         <ParsingHeaderActionTemplate
-          handleSchemaUpload={(schema) => setSchemaValue(schema)}
+          handleSchemaUpload={(schema: unknown) => setSchemaValue(schema)}
```

Comment:
can we use something specific rather than unknown

Reply:
assign to unit tester